API - JSON:API chapter (#136)
infinum/rails-handbook: API/JSON:API.md

> JSON:API is a specification for tailoring APIs that respond in JSON. Its purpose is to take the API `design` out of the equation so that developers can focus on API `delivery`. Due to its structure, tooling can be built around the specification which handles the bulk of work regarding parsing requests and formatting responses.

Reviewer: ```suggestion
JSON:API is a specification for tailoring APIs that respond with JSON. Its purpose is to take the API `design` out of the equation so that developers can focus on API `delivery`. Due to its structure, tooling can be built around the specification for handling the bulk of the work regarding parsing requests and formatting responses.
```

> If your work consists of building or working with JSON:API APIs, read the [specification](https://jsonapi.org/) to understand the concepts behind it. If you're interested in what the community has to say about JSON:API, check these blog posts out:

Reviewer: ```suggestion
If your work consists of building or working with JSON:API APIs, read the [specification](https://jsonapi.org/) to understand the concepts behind it. If you're interested in what the community has to say about JSON:API, check out these blog posts:
```

- [Standardizing RESTful JSON APIs with OpenAPI Spec](https://oozou.com/blog/standardizing-restful-json-apis-with-openapi-spec-53)
- [The Benefits of Using JSON API](https://nordicapis.com/the-benefits-of-using-json-api/)

### Standardized APIs

Developers at Infinum have implemented dozens of APIs and the [JSON:API query builder](https://github.com/infinum/jsonapi-query_builder) is a collection of the most required features a modern API has to offer.

> The gem allows a developer to use a nice DSL for the so called *query classes*, which are classes that contain JSON:API related logic for resource manipulation (filtering, sorting, pagination etc...).

Reviewer: ```suggestion
The gem allows a developer to use a nice DSL for the so-called *query classes*, which are classes that contain JSON:API related logic for resource manipulation (filtering, sorting, pagination etc...).
```

> If you're just starting a project, install the gem and make your life easier.

Reviewer: ```suggestion
If you're just starting a new project, install the gem and make your life easier.
```

JSON:API is a specification for formatting JSON APIs. Its purpose is to take API format design out of the equation so that you can focus on delivering your API. Due to its structure, tooling can be built around the specification which handles the bulk of work regarding parsing requests and formatting responses.

> The specification can be found [here](https://jsonapi.org/). If your work will concern building or working with JSON:API APIs, read the specification to understand the concepts behind it.

Author: Should we add JSON:API gems we're using?

Reviewer 2: would add blog links to stuff like
https://oozou.com/blog/standardizing-restful-json-apis-with-openapi-spec-53
or 
https://nordicapis.com/the-benefits-of-using-json-api/

I had developers (ok, one developer) who did not know how to read the official documentation and we needed to explain it to him in a not so "documenty" way.

Reviewer 3: I like both ideas 👍 

But I'll add them as a todo in productive and once we have all chapters more or less approved and deployed, we can then see everything as a whole and have a clearer picture where to fill in additional info 🙂
